### DISCUSSION TITLE
How do I decipher these error messages?

### DISCUSSION THREAD

**Discussion by sam-wheat** [AvaloniaUI/Avalonia#8390] [Q&A]
> [Binding] Error in binding to 'Avalonia.Xaml.Interactions.Core.DataTriggerBehavior'.'Binding': 'Null value in expression '{empty}' at ''.' (DataTriggerBehavior #28378434)

    <i:Interaction.Behaviors>
		<ia:DataTriggerBehavior Binding="{Binding $parent[local:GroupHeaderCell], Path=IsExpanded}" ComparisonCondition="Equal" Value="true">
			<ia:ChangePropertyAction TargetObject="{Binding #btn}" PropertyName="Content" Value="&#xE936;"/>
			
		</ia:DataTriggerBehavior>
		<ia:DataTriggerBehavior Binding="{Binding $parent[local:GroupHeaderCell], Path= IsExpanded}" ComparisonCondition="Equal" Value="false">
			
			<ia:ChangePropertyAction TargetObject="{Binding #btn}" PropertyName="Content" Value="&#xE937;"/>
		</ia:DataTriggerBehavior>
	</i:Interaction.Behaviors>

The datatrigger appears to work - why the error? The (only) reason I know the error msg applies to this XAML is because this is the only `DataTriggerBehavior` in my app.

> [Binding] Error in binding to 'Avalonia.Xaml.Interactions.Core.InvokeCommandAction'.'Command': 'Null value in expression '{empty}' at ''.' (InvokeCommandAction #19241577)

> [Binding] Error in binding to 'Avalonia.Xaml.Interactions.Core.InvokeCommandAction'.'CommandParameter': 'Null value in expression '{empty}' at ''.' (InvokeCommandAction #19241577)

> [Binding] Error in binding to 'Avalonia.Xaml.Interactions.Core.InvokeCommandAction'.'Command': 'Null value in expression '{empty}' at ''.' (InvokeCommandAction #15063922)

> [Binding] Error in binding to 'Avalonia.Xaml.Interactions.Core.InvokeCommandAction'.'Command': 'Null value in expression '{empty}' at ''.' (InvokeCommandAction #12218139)

No idea where to look for these errors.  Filename and  line number in the XAML file would be helpful.

**Comment by maxkatz6** [comment]
> The datatrigger appears to work - why the error

I assume, this error is logged before control is attached to the visual tree. I.e., when it's created without inherited datacontext yet. CC @wieslawsoltes
